Related/component/ProductInformation.jsx

```diff
@@ -52,38 +48,43 @@ class ProductInformation extends React.Component {
         }
     }
 
-    showModal() {
-      this.setState({show: true});
-    }
-
-    hideModal() {
-      this.setState({show: false});
-    }
-
     render() {
         let displayPrice;
         if (this.state.salePrice === null) {
-            displayPrice = <div class={styles.price}>${this.state.defaultPrice}</div>;
+            displayPrice = <div class={styles.defaultPrice}>${this.state.defaultPrice}</div>;
```

Comment:
```suggestion
            displayPrice = <div className={styles.defaultPrice}>${this.state.defaultPrice}</div>;
```
Need className not class

Reply:
Updated to className